src/stravalib/client.py

```diff
@@ -468,7 +468,7 @@ def get_athlete_clubs(self) -> list[model.Club]:
         # most 30 clubs are returned!
```

Comment:
I see this comment here. it is true that if someone is a member of more then 30 clubs, it will only return 30. should we fix this now?